### DISCUSSION TITLE
Could not find generator 'flipper:active_record'

### DISCUSSION THREAD

**Discussion by amitmahajan12** [jnunemaker/flipper#591] [Q&A]
I added the following to my `Gemfile`:
```
gem 'flipper'
gem 'flipper-active_record'
```

Then did `bundle install`, but after that `rails g flipper:active_record` doesn't work with the error `Could not find generator 'flipper:active_record'`

Any idea what the issue could be?

**Comment by jnunemaker** [comment]
Can you paste your Gemfile.lock or what versions of rails/etc you are using?
